Havit.Blazor.Components.Web.Bootstrap/Chips/HxChipList.razor.cs

```diff
@@ -64,6 +64,15 @@ static HxChipList()
 	/// Called when chip remove button is clicked.
 	/// </summary>
 	[Parameter] public EventCallback<ChipItem> OnChipRemoveClick { get; set; }
+
+	[Parameter] public EventCallback<ChipItem> OnResetClick { get; set; }
+
+	[Parameter] public bool Reset { get; set; }
```

Comment:
Should I add the `Settings` + `Defaults` to be able to pre-set these?

Reply:
@hakenr yes please.